share/xtp/xml/qmmm.xml

```diff
@@ -6,6 +6,8 @@
     <map_file help="xml file with segment definition" default="votca_map.xml" />
     <job_file help="name of jobfile to which jobs are written" default="qmmm_jobs.xml" />
     <io_states help="states to write jobs for and which to parse from jobfile" default="n e h" />
+    <segments help="segments to write jobs for and which to parse from jobfile" default="all"/>
+    <use_gs_for_ex help="If true uses the ground state geometry for excited states" default="false"/>
```

Comment:
```suggestion
<io_jobfile help="options to write a jobfile">
    <states help="states to write jobs for and which to parse from jobfile" default="n e h" />
    <segments help="segments to write jobs for and which to parse from jobfile" default="all"/>
    <use_gs_for_ex help="If true uses the ground state geometry for excited states" default="false"/>
  </io_jobfile>
```
I think this would be clearer. What do you think?